animation_nodes/algorithms/perlin_noise.pyx

```diff
@@ -26,6 +27,54 @@ def wiggleVectorList(amount, double evolution, amplitude, int octaves, double pe
         values[i] = perlinNoise1D(evolution + i * 354623, persistance, octaves) * _amplitude[i % 3]
     return result
 
+def wiggleFloatList(amount, double evolution, amplitude, int octaves, double persistance):
+    cdef FloatList result = FloatList(length = amount)
+    cdef float *values = <float*>result.data
+    cdef Py_ssize_t i
+    for i in range(amount):
+        values[i] = perlinNoise1D(evolution + i * 354623, persistance, octaves) * amplitude
+    return result
+
+def wiggleEulerList(amount, double evolution, amplitude, int octaves, double persistance):
+    cdef EulerList result = EulerList(length = amount)
+    cdef Euler3 *values = <Euler3*>result.data
+    cdef float _amplitude[3]
```

Comment:
Why is this needed? Just use the components directly.

Reply:
I kept my code as close to wiggleVectorList as possible; it does this, and I assumed it was a cython detail. It does appear to work fine without, so I'll fix up my changes.

Reply:
I think the Vector Wiggle code needed this because of the `_amplitude[i % 3]` part, so it is not needed here.